assignments/arrays.js

```diff
@@ -63,32 +63,33 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let myCar = inventory.filter(car => car.id === 33)[0];
+console.log(`Car 33 is a` + " " + myCar.car_year + " " +  myCar.car_make + " " + myCar.car_model);
```

Comment:
```suggestion
console.log(`Car 33 is a ${myCar.car_year} ${myCar.car_make} ${myCar.car_model}`);
```
The way you have it is fine, this is just another way to do it. Template literals were introduced in ES6 to make using expressions in strings easier. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals